pandaharvester/harvestersubmitter/htcondor_submitter.py

```diff
@@ -105,27 +112,44 @@ def parse_batch_job_filename(value_str, file_dir, batchID):
 class HTCondorSubmitter(PluginBase):
     # constructor
     def __init__(self, **kwarg):
-        self.nProcesses = 1
         self.logBaseURL = None
         PluginBase.__init__(self, **kwarg)
         # template for batch script
         tmpFile = open(self.templateFile)
         self.template = tmpFile.read()
         tmpFile.close()
         # number of processes
-        if self.nProcesses < 1:
-            self.nProcesses = None
+        try:
+            self.nProcesses
+        except AttributeError:
+            self.nProcesses = 1
+        else:
+            if (not self.nProcesses) or (self.nProcesses < 1):
+                self.nProcesses = 1
 
     # submit workers
     def submit_workers(self, workspec_list):
         tmpLog = core_utils.make_logger(baseLogger, method_name='submit_workers')
         tmpLog.debug('start nWorkers={0}'.format(len(workspec_list)))
+        # get info by cacher in db
```

Comment:
I briefly chatted with Fernando yesterday. Is it possible to take queue's corecount and maxrss in simple_worker_maker instead to set workSpec.nCore and minRamCount if len(jobspec_list) ==0
at ~ https://github.com/PanDAWMS/panda-harvester/blob/master/pandaharvester/harvesterworkermaker/simple_worker_maker.py#L15 ?
So this wouldn't have to implemented in each submitter plugin.